typing of version 3.0.0-beta.8 is broken

## Expected Behavior

version 3.0.0-beta.8 should be usable in TypeScript

## Current Behavior

The compilation fails due to what seems to be an error in the chart.js typings:

```
Error: node_modules/chart.js/dist/chart.esm.d.ts:197:7 - error TS7051: Parameter has a name but no type. Did you mean 'arg0: ChartArea'?

197  draw(ChartArea): void;
```

## Environment
<!-- Include as many relevant details about the environment you experienced the bug in -->
* Chart.js version: 3.0.0-beta.8
* Browser name and version: NA
* Link to your project: NA
* TypeScript version: 4.0.5

--- a/types/layout.d.ts
+++ b/types/layout.d.ts
@@ -47,7 +47,7 @@ export interface LayoutItem {
 	/**
 	 * Draws the element
 	 */
-	draw(ChartArea: ChartArea): void;
+	draw(chartArea: ChartArea): void;
 	/**
 	 * Returns an object with padding on the edges
 	 */

--- a/types/layout.d.ts
+++ b/types/layout.d.ts
@@ -47,7 +47,7 @@ export interface LayoutItem {
 	/**
 	 * Draws the element
 	 */
-	draw(ChartArea): void;
+	draw(ChartArea: ChartArea): void;
 	/**
 	 * Returns an object with padding on the edges
 	 */

